### ISSUE TITLE
main branch frontend didn't compile

### ISSUE THREAD

**Issue by RyanLi1028** [Get-It-Done-403/Get-It-Done#48]
after running `npm install` and `npm start`, the frontend webpage showed the following error:
```
Failed to compile.

[eslint] 
src/firebase.js
  Line 11:5:  'REMOVED' is not defined  no-undef

Search for the keywords to learn more about each error.
ERROR in [eslint] 
src/firebase.js
  Line 11:5:  'REMOVED' is not defined  no-undef

Search for the keywords to learn more about each error.

webpack compiled with 1 error
```

**Comment by what-it-izz** [comment]
I've encountered the same issue.

**Comment by what-it-izz** [comment]
I've made the application usable on my end by reverting the file `firebase.js` back to this instance of it:
[https://github.com/Get-It-Done-403/Get-It-Done/blob/f1631de0a332de6b8ca52842281ed168b1ae0493/frontend/get-it-done-frontend/src/firebase.js ]( https://github.com/Get-It-Done-403/Get-It-Done/blob/f1631de0a332de6b8ca52842281ed168b1ae0493/frontend/get-it-done-frontend/src/firebase.js )

Was the firebaseConfig was supposed to be removed? Was I supposed to initialize it with something else? I tried using the  service key shared on slack but that didn't work.
